src/main/resources/META-INF/plugin.xml

```diff
@@ -211,6 +211,22 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.CheckoutNextAction"
+                class="com.virtuslab.gitmachete.frontend.actions.navigation.CheckoutNextAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt DOWN" />
```

Comment:
Why do we need Alt (option on macOS)? Cannot it be just arrows? Navigation in Git Log works with arrows only.

Reply:
Right, good point... if it arrows truly aren't available (and/or have some unintended side effects), let's put a comment here why.

I assume the problem might lie in that Git Machete tab isn't truly _focusable_ in the same way as Log tag is... but maybe I'm wrong.

Generally, we need to avoid at all cost a scenario where a branch is checked out **inadvertently**: a user thinks that the focus is on e.g. editor and clicks an arrow, and then suddenly a different branch checks out. Note that arrows in Log tab don't cause any **side effects**, unlike with Git Machete tab.

Reply:
okay then. so I would make arrows navigate the layout without the checkout but checkout on `Enter` key. wdyt @vzmerr , @PawelLipski ?

Reply:
what if the user does not notice that the focus is on the machete tab, and presses enter while a branch is highlighted, assuming it would add a new line in the editor?

Reply:
ofc there is this risk 😕 creating a warning popup before the action will make the process of checkouts less convenient (unless we provide `dont show it again` option). 

on the other hand 
1. the selection indication is rather clear and hard to miss
2. if the checkout is not possible (uncommitted changes) if will fail anyway
3. if the checkout is possible one can always checkout the previous branch back with the same ease painlessly learning on their own mistake

Reply:
Huh TBH this is a pretty risky idea, exactly due to

> what if the user does not notice that the focus is on the machete tab, and presses enter while a branch is highlighted, assuming it would add a new line in the editor?

My suggestion would be to first check if there is even anything like _focus on machete tab_ 🤔 I suppose that we can currently only distinguish between _machete tab shown_ and _machete tab hidden_

Reply:
> what if the user does not notice that the focus is on the machete tab, and presses enter while a branch is highlighted, assuming it would add a new line in the editor?

I agree there is a such risk but tell the cost is minimal

Reply:
>  it is more clearly indicated in Git Log where additionaly you have the row selection
Awww indeed I see now 🤦🏻 

Okay... arrows let it be then! @vzmerr pls update plugin.xml, change notes and the docs... and I think we can finally merge this PR to `develop` (and thus also meld v3.0.4 into v3.1.0, i.e. the next release will be v3.1.0).

Reply:
gee, I just remembered that assigning any pre-existing key combination to an IntelliJ plugin action, with or without focus has no effect. This means that, for example, because ctrl+c is preassigned, despite the fact that it does nothing in the machete tab, assigning it, for example, to an action for copying a branch, would not have any effect. this is the same for the arrows. Sorry for not mentioning it earlier under this thread.

Reply:
Pls verify experimentally that indeed pure arrows (no Alt/Option) will have no effect even when Git Machete tab is verified

Reply:
yes, alt + arrows are indeed assigned. So I would try the pure arrows on focus and let you know.

Reply:
I confirm that using pure arrows for navigation has **no** effect, even if machete tab has the focus.